### DISCUSSION TITLE
多开分区模块失效/无法保存配置解决方法

### DISCUSSION THREAD

**Discussion by w311ang** [LSPosed/LSPosed#1928] [Show and tell]
## 原因
lsp用来保存模块配置的文件夹`/data/misc/[UUID]/prefs*`的错误权限`u:object_r:system_data_file:s0`被selinux阻止导致

## 解决办法
MT管理器运行，终端要su后去除sudo运行
```bash
sudo chcon -R u:object_r:magisk_file:s0 /data/misc/*-*-*-*-*
```

## 复现问题
```bash
sudo chcon -R u:object_r:system_data_file:s0 /data/misc/*-*-*-*-*
```

## 受影响的模块
* 源计划(原应用伪装)

## Tips
`sudo /system/bin/ls -Z`查看文件(夹)selinux权限

**Comment by yujincheng08** [comment]
@vvb2060

**Comment by vvb2060** [comment]
https://github.com/LSPosed/LSPosed/pull/1929
